libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -164,6 +164,7 @@ singleton AP_Vehicle method start_takeoff boolean float (-LOCATION_ALT_MAX_M*100
 singleton AP_Vehicle method set_target_location boolean Location
 singleton AP_Vehicle method get_target_location boolean Location'Null
 singleton AP_Vehicle method set_target_velocity_NED boolean Vector3f
+singleton AP_Vehicle method set_target_angle_and_climbrate boolean float -90 90 float -180 180 float -360 360 float -FLT_MAX FLT_MAX boolean float -FLT_MAX FLT_MAX
```

Comment:
so close!  I think the roll limits should be +-180 and pitch should be +-90.  I think it's reversed here.

Reply:
oops, fixed, hopefully I will get there in the end